Tools/autotest/arducopter.py

```diff
@@ -5314,12 +5314,17 @@ def fly_gyro_fft_harmonic(self):
                 "SIM_GYR1_RND": 20,  # enable a noisy gyro
             })
             # motor peak enabling FFT will also enable the arming
-            # check, self-testing the functionality
+            # check, self-testing the functionality.
             self.set_parameters({
                 "FFT_ENABLE": 1,
                 "FFT_MINHZ": 50,
                 "FFT_MAXHZ": 450,
                 "FFT_SNR_REF": 10,
+                # filter setup with zero attenuation
```

Comment:
I will have to think about this :)

Reply:
i really don't like relying on the value of parameters that are hidden as they are disabled. It leads to unpredictable behaviour. A user log that relied on it could not be analysed properly as we don't know the value it uses

Reply:
I agree, but instead we should just use the default harmonic value of 7. Having to have the notch enabled with 0 attenuation to see the effect of this is just not useful. I have helped people debug the effect of the FFT when they have the throttle notch enabled - we couldn't do this in this instance if they have to use the FFT to drive the notch to see the effect.

Reply:
fixed as discussed